compiler/rustc_hir/src/def.rs

```diff
@@ -218,6 +218,13 @@ impl DefKind {
             | DefKind::Impl => None,
         }
     }
+
+    pub fn is_macro(self) -> bool {
+        match self {
+            DefKind::Macro(..) => true,
+            _ => false,
+        }
+    }
```

Comment:
I don't think this method is necessary, it's trivial `matches!(kind, DefKind::Macro(..))` and it's not commonly used.

Reply:
I personally think `!matches!()` looks atrocious. That said, fair enough.